src/game/input.c

```diff
@@ -63,16 +58,11 @@ void Input_CheckConflicts(INPUT_LAYOUT layout_num)
             if (role1 == role2) {
                 continue;
             }
-
             INPUT_SCANCODE scancode2_user =
                 Input_GetAssignedScancode(layout_num, role2);
-
```

Comment:
`scancode1_user` and `scancode2_user` can be renamed to just `scancode1` and `scancode2` now.